sqldelight-runtime/build.gradle

```diff
@@ -12,6 +12,7 @@ kotlin {
     commonMain {
       dependencies {
         implementation deps.kotlin.stdlib.common
+        implementation "co.touchlab.stately:stately:0.3.6-k1.3.0-g4.10.2-a1"
```

Comment:
These version strings are crazy, but until we have a more stable publish situation, it's just easier to visually see what's happening. I also haven't set up bintray to accept snapshots. TODO.